src/Mod/Part/App/OpenCascadeAll.h

```diff
@@ -115,12 +115,12 @@
 
 #include <BRepAlgo.hxx>
 #include <BRepAlgo_NormalProjection.hxx>
-#include <BRepAlgoAPI_BooleanOperation.hxx>
-#include <BRepAlgoAPI_Common.hxx>
-#include <BRepAlgoAPI_Cut.hxx>
+#include <FCRepAlgoAPI_BooleanOperation.hxx>
+#include <FCRepAlgoAPI_Common.hxx>
+#include <FCRepAlgoAPI_Cut.hxx>
 # include <BRepAlgoAPI_Defeaturing.hxx>
-#include <BRepAlgoAPI_Fuse.hxx>
-#include <BRepAlgoAPI_Section.hxx>
+#include <FCRepAlgoAPI_Fuse.hxx>
+#include <FCRepAlgoAPI_Section.hxx>
 

```

Comment:
This header file is wrong, file ending incorrect. Doesnt affect compilation though, is this never used?

Reply:
I can't find any occurrence of the `OpenCascadeAll.h` string in the code.

Reply:
I revrted the file to how it used to be - this shouldn't have been changed in the first place, regexp run out of scope.

Reply:
@FlachyJoe it's only used when using precompiled headers on MSVC builds -- fixed it now